components/search_results_item/search_results_item.jsx

```diff
@@ -40,6 +42,11 @@ class SearchResultsItem extends React.PureComponent {
         */
         post: PropTypes.object,
 
+        /**
```

Comment:
nit: alignment relative to other comments, but I'm wondering if we can just omit these semi-useless comments.

Reply:
I would love to do that, but as a separate PR, this PR is already complex enough to add cosmetic changes.